read-fonts/src/tables/layout/script.rs

```diff
@@ -0,0 +1,269 @@
+//! Additional support for working with OpenType scripts.
+
+use super::{ScriptList, Tag};
+use std::ops::Deref;
+
+/// A script chosen from a set of candidate tags.
+///
+/// Returned by the [`ScriptList::select`] method.
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub struct SelectedScript {
+    /// The actual OpenType tag of the chosen script.
+    pub tag: Tag,
+    /// Index of the script in the [`ScriptList`].
+    pub index: u16,
+    /// True if a script was chosen that wasn't in the requested list.
+    pub is_fallback: bool,
```

Comment:
Speculation: I wonder if a Rusty SelectedScript should be an enum instead?

Reply:
That's an equivalent representation but an enum with all variants carrying the same payload where the variant itself is secondary information is often awkward to use. In other words, we'll always want the tag/index but might occasionally care if we're using a fallback so it doesn't make sense to primarily categorize on that aspect.